src/kvstore/NebulaStore.cpp

```diff
@@ -104,35 +107,72 @@ void NebulaStore::loadPartFromDataPath() {
           continue;
         }
 
-        // Load raft peers info which persisted to local engine.
-        // If the partition was in balancing process before restart, we should keep it
-        // though the part is not in the meta.
         auto engine = newEngine(spaceId, path, options_.walPath_);
         std::map<PartitionID, Peers> partRaftPeers;
-        for (auto& [partId, raftPeers] : engine->allPartPeers()) {
-          bool isNormalPeer = true;
-
-          Peer raftPeer;
-          bool exist = raftPeers.get(raftAddr_, &raftPeer);
-          if (exist) {
-            if (raftPeer.status != Peer::Status::kNormalPeer) {
-              isNormalPeer = false;
+
+        // load balancing part info which persisted to local engine.
+        for (auto& [partId, raftPeers] : engine->balancePartPeers()) {
```

Comment:
sorry, I don't understand, what does balancePartPeers mean, shouldn't we load allPartPeers?

Reply:
I known, ignore it

Reply:
That interface has been removed~